jvm/src/main/scala/com/avast/sst/jvm/execution/ConfigurableThreadFactory.scala

```diff
@@ -10,7 +10,9 @@ import com.avast.sst.jvm.execution.ConfigurableThreadFactory.Config
 /** Thread factory (both [[java.util.concurrent.ThreadFactory]] and [[java.util.concurrent.ForkJoinPool.ForkJoinWorkerThreadFactory]])
   * that creates new threads according to the provided [[com.avast.sst.jvm.execution.ConfigurableThreadFactory.Config]].
   */
-class ConfigurableThreadFactory(config: Config) extends ThreadFactory with ForkJoinWorkerThreadFactory {
+class ConfigurableThreadFactory(config: Config, uncaughtExceptionHandler: UncaughtExceptionHandler = LoggingUncaughtExceptionHandler)
```

Comment:
Let's move this change (which is backwards incompatible) to a separate PR. Thanks.